lightning/src/ln/monitor_tests.rs

```diff
@@ -2228,8 +2228,10 @@ fn test_anchors_aggregated_revoked_htlc_tx() {
 	let spendable_output_events = nodes[0].chain_monitor.chain_monitor.get_and_clear_pending_events();
 	assert_eq!(spendable_output_events.len(), 2);
 	for (idx, event) in spendable_output_events.iter().enumerate() {
-		if let Event::SpendableOutputs { outputs } = event {
+		if let Event::SpendableOutputs { outputs, channel_id } = event {
 			assert_eq!(outputs.len(), 1);
+			// let funding_outpoint = OutPoint { txid: chan_a.3.txid(), index: 0 };
+			// assert_eq!(channel_id, &funding_outpoint.to_channel_id());
```

Comment:
@tnull this bit here is failing, whats the best way to construct the channel_id on the right side of the assertion?